packages/editor/src/components/rich-text/remove-browser-shortcuts.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { rawShortcut } from '@wordpress/keycodes';
+import { KeyboardShortcuts } from '@wordpress/components';
+
+export function RemoveBrowserShortcuts() {
+	return (
+		<KeyboardShortcuts
+			bindGlobal
+			shortcuts={ {
+				[ rawShortcut.primary( 'z' ) ]: () => false,
```

Comment:
This feels a bit like we're relying a bit too much on incidental support of the `false` return offered as a "convenience" by Mousetrap, but otherwise not the canonical way for preventing default behavior:

https://craig.is/killing/mice#api.bind.default

Reply:
> This feels a bit like we're relying a bit too much on incidental support of the `false` return offered as a "convenience" by Mousetrap, but otherwise not the canonical way for preventing default behavior:
> 
> https://craig.is/killing/mice#api.bind.default

Improved in 41ed725.

Reply:
Thanks!